### PR TITLE
epic/3584

### PR DESCRIPTION
**Pull request by nocturnalastro**
Some times a second name is required to conform to formats external to udes

User-story: 3599
Signed-off-by: Michele Costa <michele.costa@unipart.io>

### CODE REVIEW

**Comment by parisiale** [comment]
👍

File: addons/udes_stock/models/stock_move.py [hunk 1]
```diff
@@ -451,31 +451,40 @@ def refactor_action_group_by_move_line_key(self):
...
         picking_type = self.mapped('picking_type_id')
         picking_type.ensure_one()
 
         if not picking_type.u_move_line_key_format:
             return
 
-        pickings = self.mapped('picking_id')
         mls_by_key = self.mapped('move_line_ids').group_by_key()
 
+        return self.refactor_by_move_line_groups(mls_by_key.items())
```
Comment (ws-mps) [inline addons/udes_stock/models/stock_move.py]: For consistency, can we do the same at `refactor_action_group_by_move_key` ?

Comment (nocturnalastro) [reply addons/udes_stock/models/stock_move.py]: Sure :)

**Comment by parisiale** [comment]
Kudos for "refactor the refactoring"!!!

**Pull request closed without merging**